docs/projected-volume.md

```diff
@@ -0,0 +1,27 @@
+# Projected Volumes and Windows Pods
+
+A [projected volume](https://kubernetes.io/docs/concepts/storage/volumes/#projected) allows mapping of several existing
+volume sources into the same directory in the container. With the [proposal for file permission handling in projected
+service account volume](https://github.com/kubernetes/enhancements/pull/1598) enhancement, the projected files have
+the correct permissions set including container user ownership for Linux pods when `RunAsUser` is set in the Pod 
+`SecurityContext`. However, for Windows pods this is not the case when `RunAsUsername` is set, due to differences in 
+the way user accounts are managed in Windows. Windows stores and manages local user and group accounts in a database
+file called Security Account Manager (SAM). This database is not shared between the Windows container host and the
+running containers. This prevents the kubelet from setting correct ownership on the files in the projected volume.
+This problem is tracked by [Windows Pod with RunAsUserName and a Projected Volume does not honor file
```

Comment:
What permissions will the files have and what owner will the files have?

Reply:
I have added what the default permissions will look like. 

>This implies all administrator users like `ContainerAdministrator` will have read, write and execute access while,
non-administrator users will have read and execute access.